app/services/api/webflow-api.types.ts

```diff
@@ -130,6 +130,7 @@ export interface RawSpeaker extends Item {
   title?: string
   twitter?: string
   externalURL?: string
+  website?: string
```

Comment:
I'm guessing this was an addition on the webflow side?

Reply:
yes. we have a miro board that has all the fields as they're being add but this was missed